README.md

```diff
@@ -1,62 +1,57 @@
-:wave:	*New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:	
-
+:wave: *New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:
 
 ![Node.js CI](https://github.com/openmrs/openmrs-esm-patient-management/workflows/Node.js%20CI/badge.svg)
 
-
 # OpenMRS Patient Management
 
-This is a [Lerna](https://lerna.js.org/) project containing patient management frontend modules which deal with creating, searching, and listing patients. This package handles the following:
+This repository bundles together frontend modules for the OpenMRS SPA. These modules handle concerns relating to registering and editing patients, searching for existing patients, creating and managing patient lists and managing patient queues in an outpatient setting. The microfrontends within this module include:
```

Comment:
```suggestion
This repository contains frontend modules for the OpenMRS SPA. These modules relate to registering and editing patients, searching for existing patients, creating and managing patient lists and managing patient queues in an outpatient setting. The modules within this repository include:
```

"bundles together" sounds like it's doing some kind of action, like its a bundler